### ISSUE TITLE
Contact us/ Form: A11y and UI styling

### ISSUE THREAD

**Issue by yesenia-gtz** [Office-of-Digital-Services/California-State-Web-Template-Website#952]
**A11y:**
- [x] Current heading structures are H1 and H3. Need to add an H2
- [ ] Presentational element is exposed to assistive. Decide whether the element is decorative or meaningful and make sure it's marked up correctly. technologies. iFrame is missing a title/label
![image (1)](https://user-images.githubusercontent.com/98193284/207421957-ffbdebf2-f21a-4aa0-b602-b2d4acfbb3c1.png)

- [x] Link text used for multiple different destinations. Use the same exact description for the text links.
![image](https://user-images.githubusercontent.com/98193284/207421910-a26d2220-233c-4fc9-bbd2-d298a6d968c9.png)

-----------------------

**Styling:**

- [x] The styling for the dropdown is different from the Figma. Can this be adjusted? 
![Screen Shot 2022-12-13 at 10 55 57 AM](https://user-images.githubusercontent.com/98193284/207420733-5b5a0f54-876e-4a85-b034-f293f4007809.png)

- [x] Looks like the error message design fixes still need to be addressed:
Figma
![image](https://user-images.githubusercontent.com/98193284/207421120-392e8640-246e-4f84-aab6-ff466019c1d5.png)
Site
![image (1)](https://user-images.githubusercontent.com/98193284/207421137-f7239be4-c7d6-4d90-9737-21c32f48dba7.png)

**Comment by kkoryaka** [comment]
@sharon-rabang this is now updated https://as-cdt-pub-migr-betaws-ww-p-001-stage.azurewebsites.net/contact-us.html

**Comment by kkoryaka** [comment]
cc @extinah

**Comment by pglomboy** [comment]
@kkoryaka @sharon-rabang after selecting from the drop down, the functionality of changing your option has been disabled 
![image](https://user-images.githubusercontent.com/120594656/208772497-658c3f46-4a90-43cc-9c28-a16de431c154.png)

**Comment by pglomboy** [comment]
Item: Link text used for multiple different destinations. Use the same exact description for the text links. has been resolved. This item still needs to be resolved:Presentational element is exposed to assistive. Decide whether the element is decorative or meaningful and make sure it's marked up correctly. technologies. iFrame is missing a title/label @kkoryaka

**Comment by pglomboy** [comment]
And we still need to fix the heading structures. Currently at: H1,H2,H2,H2 and H4. Need and H3 @kkoryaka @sharon-rabang

**Comment by yesenia-gtz** [comment]
@kkoryaka @carterm tagging for visibility

**Comment by sharon-rabang** [comment]
@kkoryaka @pglomboy @extinah See our UX design feedback in #525 for reference.

@pglomboy 
- The "Main role" combo box is changing to a dropdown, so the caret will function 
- For the H4, please talk with dev

**Comment by kkoryaka** [comment]
@pglomboy a11y should be fixed except that recapcha. It's third party and we can't do anything about it.

**Comment by yesenia-gtz** [comment]
closing
